source/includes/steps-fle-convert-to-a-remote-master-key-kmip.yaml

```
@@ -1,20 +1,20 @@
-title: Configure Your KMIP KMS
-ref: configure-kmip-kms
+title: Configure Your KMIP Provider
+ref: configure-kmip-provider
 content: |
 
-  To connect a MongoDB driver client to your KMIP KMS, you must configure your KMS
-  such that it accepts your client's TLS certificate.
+  To connect a MongoDB driver client to your KMIP provider, you must configure
+  your KMIP provider such that it accepts your client's TLS certificate.
 
-  Consult the documentation for your KMIP KMS for information on how
-  to accept your client certificate with your specific KMS.
+  Consult the documentation for your KMIP provider for information on how
+  to accept your client certificate with your specific KMIP provider.
```

I know it's existing text, but now that I read it again it's very redundant.

```suggestion
  to accept your client certificate.
```

---

Yup agree this is clear without the repetition